src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -0,0 +1,45 @@
+import isEmpty from 'ramda/src/isEmpty';
+import { STORY_PAGE } from '#app/routes/utils/pageTypes';
+import getAssetType from './getAssetType';
+import fetchPageData from '../../utils/fetchPageData';
+
+const pageTypeUrls = (assetType, service) => {
+  switch (assetType) {
+    case STORY_PAGE:
+      return [
+        {
+          name: 'mostRead',
+          path: `/${service}/mostread`,
+        },
+      ];
+    default:
+      return null;
+  }
+};
+
+const validateResponse = ({ status, json }, name) => {
+  if (status === 200 && !isEmpty(json)) {
```

Comment:
Just thinking about status codes will the most read endpoint always return 200 in the success case? One case I can think of is a 202 response on the first request for most-read for a service; will the first cache of a page have a missing 'most read' section? Once the page cache expires this situation would be resolved as the next call to most-read would issue a 200 with the content. This endpoint may not 202, just asking if we know this and if so, are we okay with the above?

Reply:
not sure on the status codes actually, will check with the ares team. however if the first request was to return a non 200 we will simply not append the newly fetched data to the page data, and in the case of mostRead it will fallback to its client side data fetch using `useEffect()`

Reply:
Hmm, if we fallback to `useEffect` I think we'd experience a big traffic spike to the endpoint to the most read endpoint until the initial page cache expires? I imagine that's okay, but just making sure we've thought this through.

Reply:
Thats how mostRead has been built. MostRead is current client side only.